cdap-api/src/main/java/co/cask/cdap/api/artifact/ArtifactManager.java

```diff
@@ -35,6 +35,14 @@
    */
   List<ArtifactInfo> listArtifacts() throws IOException;
 
+  /**
+   * Get the list of artifacts in the repository in the specified namespace and system namespace
+   *
+   * @return {@link List<ArtifactInfo>} list of artifact info
```

Comment:
This would link to what? The List interface?

Reply:
i think we can remove this, was from copy-paste of the existing one.

Reply:
I'm missing a param in the javadoc, will fix this